packages/wallet/src/__stories__/b-dummy-wallet-states.ts

```diff
@@ -3,7 +3,7 @@ import * as fundingStates from '../redux/channel-state/funding/state';
 import * as sharedStates from '../redux/channel-state/shared/state';
```

Comment:
Is the plan to remove this file too at some point?

Reply:
Yes, we should remove that file once indirect funding B player scenario is set up.